Source/Core/VideoCommon/AVIDump.cpp

```diff
@@ -66,7 +66,7 @@ bool AVIDump::Start(HWND hWnd, int w, int h)
 	if (SConfig::GetInstance().m_SYSCONF->GetData<u8>("IPL.E60"))
 		s_frame_rate = 60; // always 60, for either pal60 or ntsc
 	else
-		s_frame_rate = VideoInterface::TargetRefreshRate; // 50 or 60, depending on region
+		s_frame_rate = (int) VideoInterface::TargetRefreshRate; // 50 or 60, depending on region
```

Comment:
That extra space between the cast and the value looks a bit odd to me...shouldn't this be without a space?